macros/sql/surrogate_key.sql

```diff
@@ -1,4 +1,8 @@
 {%- macro surrogate_key(field_list) -%}
+    {{ return(adapter.dispatch('surrogate_key', packages = dbt_utils._get_utils_namespaces())(field_list, *args)) }}
+{% endmacro %}
```

Comment:
@jtcohen6 one more... thanks for bearing with us as we muddle through this!
I added an `*args` arg, but it still doesn't like it...
```
Compilation Error in model test_surrogate_key (models/sql/test_surrogate_key.sql)
  macro 'dbt_macro__surrogate_key' takes not more than 1 argument(s)
  
  > in macro surrogate_key (macros/sql/surrogate_key.sql)
  > called by model test_surrogate_key (models/sql/test_surrogate_key.sql)
```

Reply:
Ugh. Yeah I think this may need to be something like
```sql
{%- macro surrogate_key(field_list) -%}
    {% set placeholder = varargs %}
    {{ return(adapter.dispatch('surrogate_key', packages = dbt_utils._get_utils_namespaces())(field_list, *args)) }}
{% endmacro %}
```